starter-project-web-vue2/pages/dagmawit/_id.vue

```diff
@@ -0,0 +1,45 @@
+<template>
+  <div>
+    <nuxt-link to="/dagmawit/">
+      <v-btn class="ma-2" color="#1F7087" dark>
+        <v-icon dark left> mdi-arrow-left </v-icon>Back
+      </v-btn>
+    </nuxt-link>
+    <br />
+    <p></p>
+    <h1 align="center">Blog: {{ blog.title }}</h1>
+
+    <v-container
+      ><h4>{{ blog.content }}</h4></v-container
+    >
+  </div>
+</template>
+
+<script>
+import axios from 'axios'
+
+export default {
+  data() {
+    return {
+      blog: {
+        title: '',
+        content: '',
+      },
+    }
+  },
+
+  async created() {
+    try {
+      const response = await axios.get(
+        `https://blog-app-backend.onrender.com/api/articles/${this.$route.params.id}`
+      )
+      this.blog = this.response.data
+    } catch (err) {
+      this.$nuxt.error({
```

Comment:
This page crashes when it loads. it isn't fetching the blog ig

and better copy the method to a store actions and call it like `this.$store.dispatch('dagmawit/fetchblogDetail', id) `etc
also use this.$axios for requests too